<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="gameScene">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>331</height>
     </rect>
    </property>
    <widget class="QWidget" name="start">
     <widget class="QPushButton" name="pbStartGame">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Snake</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pbset">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>250</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="set">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>游戏难度</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbEasy">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>不会真有人达不到1000分吧</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbMid">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>我奶奶都能玩</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbDiffi">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>没手都行</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbsave">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="end">
     <widget class="QPushButton" name="pbTryAgain">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>重试</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>31</pointsize>
       </font>
      </property>
      <property name="text">
       <string>你寄了</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="game">
     <widget class="QLabel" name="lbscore">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>score:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
